server/app.js

```diff
@@ -44,7 +44,7 @@ app.get('/swagger.json', (req, res) => {
   res.send(swaggerSpec);
 });
 
-app.use('/api/v1/auth', userRoutes);
+app.use('/api/v2/auth', userRoutes);
```

Comment:
How is your auth endpoint `v2` and the rest `v1`?. This shouldn't be the case. For now, it would be better to have your endpoints as `v1`

Reply:
Reverted the endpoints to v1.